plugins/player-activity-tracker.js

```diff
@@ -177,7 +186,7 @@ window.plugin.playerTracker.processNewData = function(data) {
     });
 
     // skip unusable events
-    if (!plrname || !lat || !lng || !id || skipThisMessage || ![window.TEAM_RES, window.TEAM_ENL].includes(window.teamStringToId(json[2].plext.team))) {
+    if (!plrname || !lat || !lng || !id || skipThisMessage || ![window.TEAM_RES, window.TEAM_ENL].includes(plrteam)) {
```

Comment:
```suggestion
    if (!plrname || !lat || !lng || !id || skipThisMessage || ![window.TEAM_RES, window.TEAM_ENL].includes(window.teamStringToId(plrteam))) {
```

The `plrteam` contains the faction name, but TEAM_RES and TEAM_ENL contain numbers.
Without this fix, no trackers appear on the map